app/api/posts/route.ts

```diff
@@ -0,0 +1,42 @@
+import { NextRequest, NextResponse } from 'next/server';
```

Comment:
Do we need this endpoint or could we get the posts on the server side without calling it from client side?

Reply:
maybe also we can delete it. was exploring also.

Reply:
PR 11 to solve this